Examples/MAX32662/UART/main.c

```diff
@@ -133,15 +145,23 @@ int main(void)
     MXC_Delay(MXC_DELAY_SEC(1));
 
     // Initialize the UART
+#if defined(BOARD_EVKIT_V1)
     if ((error = MXC_UART_Init(WRITING_UART, UART_BAUD, MXC_UART_APB_CLK, MAP_A)) != E_NO_ERROR) {
+#else
+    if ((error = MXC_UART_Init(WRITING_UART, UART_BAUD, MXC_UART_APB_CLK, MAP_B)) != E_NO_ERROR) {
+#endif
         Shutdown_UARTS();
         Console_Init();
         printf("-->Error initializing UART: %d\n", error);
         printf("-->Example Failed\n");
         return error;
     }
 
+#if defined(BOARD_EVKIT_V1)
     if ((error = MXC_UART_Init(READING_UART, UART_BAUD, MXC_UART_APB_CLK, MAP_A)) != E_NO_ERROR) {
+#else
+    if ((error = MXC_UART_Init(READING_UART, UART_BAUD, MXC_UART_APB_CLK, MAP_A)) != E_NO_ERROR) {
```

Comment:
#if else condition seems not needed.
The function and parameters are exactly same.
To simplify code it should be deleted.

Reply:
done [1a9774e](https://github.com/Analog-Devices-MSDK/msdk/pull/469/commits/1a9774e271546f20caceccba8c82020602e777b1)